interop-cats/shared/src/main/scala/zio/interop/queue/Queue.scala

```diff
@@ -0,0 +1,25 @@
+package zio.interop.queue
+
+import cats.effect.LiftIO
+import zio.{ Runtime, UIO, ZEnv, Queue => ZioQueue }
+
+object Queue {
+  import CQueue._
+
+  type Queue[F[+_], A] = CQueue[F, A, A]
```

Comment:
I think we probably want to bring this up into the package object.

Reply:
good idea, done 👍